.env.example

```
@@ -78,7 +78,7 @@ RESERVED_SUBDOMAINS='"app","auth","docs","design","console","go","status","api",
 NEXTAUTH_URL='http://localhost:3000'
 # @see: https://next-auth.js.org/configuration/options#nextauth_secret
 # You can use: `openssl rand -base64 32` to generate one
-NEXTAUTH_SECRET=
+NEXTAUTH_SECRET=1aeor+ATMAVZmoaIhGEH0SItf74pXlg4kS41WQxzCcE=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXTAUTH_SECRET key should go before the NEXTAUTH_URL key

(UnorderedKey)